### PR TITLE
Placed scroll arrows.

### PR DESCRIPTION
**Pull request by soehnlein**
Placed arrow buttons and styled as links.

### CODE REVIEW

File: assets/stylesheets/main.css [hunk 1]
```diff
@@ -141,6 +141,71 @@ header h3.subhead {
   white-space: nowrap;
 }
 
+.scroll-arrow-left{
```
Comment (ehlertij) [inline assets/stylesheets/main.css]: `.scroll-arrow-left` and `.scroll-arrow-right` are nearly identical. I think the only difference is `margin-left`. You should make a common class that they can share that sets all of the common styles (probably just called `.scroll-arrow`).

File: index.html [hunk 2]
```diff
@@ -29,6 +29,18 @@ <h3 class="subhead">A project by Patrice Soehnlein<br>
...
      <div class="content">
       <section class="forms">
+
+        <div class="scroll-arrow-left">
+          <a href=#scroll-left>
+            <span>&#12296;</span>
+          </a>
+        </div>
+        <div class="scroll-arrow-right">
+            <a href=#scroll-right>
```
Comment (ehlertij) [inline index.html]: These lines are tabbed over one level too far.

**Comment by ehlertij** [comment]
Looks great to me! I was thinking this could be done using `position: relative` on the parent, and then using `left` and `right` styles for the arrows, but they get cut off by the edge of the container that way. I think the way you did it is probably the best way.

Fix those couple of things I mentioned and this looks good to merge! :+1:

**Comment by soehnlein** [comment]
I cleaned up the repeating code, let me know if it looks good to you.

**Comment by ehlertij** [comment]
Looks good, fix up the tabbing and I say merge it!

**Pull request merged**